doc/developers/contributing.rst

```diff
@@ -641,22 +641,42 @@ the corresponding subpackages.
 
 We expect code coverage of new features to be at least around 90%.
 
-.. note:: **Workflow to improve test coverage**
+For guidelines on how to use ``pytest`` efficiently, see the
+:ref:`pytest_tips`.
 
-   To test code coverage, you need to install the `coverage
-   <https://pypi.org/project/coverage/>`_ package in addition to pytest.
+Available test fixtures in scikit-learn
```

Comment:
I would suggest to call this section

> Writing matplotlib related tests

instead, and change the writing accordingly. Again, it's weird to mention multiple fixtures when we only have one.